flatpak/io.github.softfever.OrcaSlicer.yml

```diff
@@ -144,7 +160,7 @@ modules:
     sources:
       - type: git
         url: https://github.com/SoftFever/Orca-deps-wxWidgets
-        branch: master
+        commit: a675dbe8fd07cb5c982cdab7b93cffc9d9f05f22 
```

Comment:
Do we need to specify this since the a675dbe8fd07cb5c982cdab7b93cffc9d9f05f22 is the latest commit in master branch?

Reply:
Yes, because they should always be the same and never mismatch during building both the appimage and flatpak. Really, it's a sanity check because of CI restrictions.

Reply:
While it would normally, one of the things that a flathub upload would very likely require is dependency locking per release version. If a flatpak only bug is discovered, then you want to fix it without any unexpected or unneeded updates. Since this is the only dependency that has to be synchronized in a different way compared to all the other ones (due to CI size constraints), I figured this is the least intrusive method to this.

Reply:
> If a flatpak only bug is discovered, then you want to fix it without any unexpected or unneeded updates.

This is actually the opposite for third-party cases in my experience. It's better to avoid pointing to different commits for different build systems. The reason I'm asking to remove the commit tag is that the Orca-deps-wxWidgets repository is specifically created for OrcaSlicer. Any changes made to that repo should be caught by the CI/CD pipeline when the change is made.

If Flatpak points to an old commit, it will miss that check. The worst-case scenario is that it results in different behaviors compared to other platforms due to different versions of wxWidgets being used, which would be a nightmare to troubleshoot.

Reply:
Alright, then you may revert both tagged parts.

Reply:
I've reverted both tagged parts.

Reply:
Sorry for the slow response.
Was a bit busy in the past two weeks.

Looks good, thank you so much!